src/modules/hotkeys/i18n/en_AU.root.json

```diff
@@ -35,6 +35,15 @@
             },
             "title": "General"
         },
+        "building": {
```

Comment:
`       "building": {
            "goto": {
                "dispatchCenter": "larmcentral",
                "nextBuilding": "Nästa byggnad",
                "previousBuilding": "Föregående byggnad",
                "title": "gå till"
            },
            "title": "Byggnad"
        },`

sv_SE

Reply:
Hope you meant se_SE and not en_AU :smile:

Reply:
yup oopsie :)

Reply:
OH wait there is a typoo by my side i see now

Reply:
"dispatchCenter": "Larmcentral"